src/static/index.js

```diff
@@ -452,7 +452,8 @@ export async function buildXMLandRSS ({ config }) {
   if (!siteRoot) {
     return
   }
-  const prefixPath = config.disableRoutePrefixing ? siteRoot : config.publicPath
+  let prefixPath = config.disableRoutePrefixing ? siteRoot : config.publicPath
+  prefixPath = prefixPath.replace(/\/$/, '')
   const xml = generateXML({
     routes: config.routes.filter(d => !d.is404).map(route => ({
       permalink: `${prefixPath}/${pathJoin(route.path)}`,
```

Comment:
You could still get a double slash here if `route.path === '/'`. Here is the code for `pathJoin`: https://github.com/nozzle/react-static/blob/master/src/utils/shared.js#L6

Perhaps `trimSlashes` should be broken into two functions: `trimLeadingSlashes` and `trimTrailingSlashes`. Then we can use those everywhere?

https://github.com/nozzle/react-static/blob/master/src/utils/shared.js#L52

Reply:
Yeah, there are so many edge cases with slashes everywhere. I would much rather have all of them going through utility functions that have tests.

Reply:
Makes sense. Will update.